docs/book/src/clusterctl/commands/commands.md

```diff
@@ -1,19 +1,20 @@
 # clusterctl commands
 
-* [`clusterctl alpha rollout`](alpha-rollout.md)
-* [`clusterctl alpha topology plan`](alpha-topology-plan.md)
-* [`clusterctl backup`](additional-commands.md#clusterctl-backup)
-* [`clusterctl completion`](completion.md)
-* [`clusterctl config`](additional-commands.md#clusterctl-config-repositories)
-* [`clusterctl delete`](delete.md)
-* [`clusterctl describe cluster`](describe-cluster.md)
-* [`clusterctl generate cluster`](generate-cluster.md)
-* [`clusterctl generate provider`](generate-provider.md)
-* [`clusterctl generate yaml`](generate-yaml.md)
-* [`clusterctl get kubeconfig`](get-kubeconfig.md)
-* [`clusterctl help`](additional-commands.md#clusterctl-help)
-* [`clusterctl init`](init.md)
-* [`clusterctl move`](move.md)
-* [`clusterctl restore`](additional-commands.md#clusterctl-restore)
-* [`clusterctl upgrade`](upgrade.md)
-* [`clusterctl version`](additional-commands.md#clusterctl-version)
+* [`clusterctl alpha rollout`](alpha-rollout.md) Manages the rollout of a MachineDeployment.
```

Comment:
```suggestion
* [`clusterctl alpha rollout`](alpha-rollout.md) Manages the rollout of a Cluster API resources. Ex: MachineDeployments.
```

Reply:
@ykakarap Good suggestion. Updated to: "Manages the rollout of Cluster API resources. For example: MachineDeployments."